docs/getting-started/1200-local-dev.md

```diff
@@ -33,7 +33,7 @@ If you do, you can install `dagger` with a single command:
 brew install dagger/tap/dagger
 ```
 
-This installs `dagger` in `/opt/homebrew/bin`:
+This installs `dagger` in `/opt/homebrew/bin` for **Mac with Apple Chip** & in `/usr/local/bin` for **Mac with Intel Chip**:
```

Comment:
May that would be more accurate to say `or in /usr/local/bin` instead of `&`

Reply:
I wonder why this difference in paths.

Reply:
I see, it’s because of ARM and rosetta.

Reply:
That's a good question indeed, here's a [Reddit post about it](https://www.reddit.com/r/MacOS/comments/jw9guu/why_did_homebrew_move_from_usrlocalto_opthomebrew/). 

And here's the answer of Homebrew maintainer

> Another reason why we chose to support two separate installations for the time being: the native ARM one is just not ready yet so users may want to use the safer, more stable x86_64 installation (and the programs it serves) interchangeably with the newer, experimental ARM one where lots of formulae still break.